app/views/users/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<% provide :title, @user.name %>
+<div class="profile">
+  <% if @user.is? current_user %>
```

Comment:
Devise cung cấp helper có sẵn `current_user` để kiểm tra user hiện tại signed-in
Em xem lại chỗ này nhé